src/main/java/com/minecolonies/coremod/items/ItemSupplyCampDeployer.java

```diff
@@ -61,6 +62,12 @@
     private static final int CHECK_X_DIRECTIONS = 4;
 
     /**
+     * Our guide Book.
+     */
+    @GameRegistry.ItemStackHolder(value = "gbook:guidebook", nbt = "{Book:\"minecolonies:book/minecolonies.xml\"}")
+    public static ItemStack guideBook;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static guideBook" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1444)